ogr/services/pagure/pull_request.py

```diff
@@ -235,3 +235,43 @@ def merge(self) -> "PullRequest":
     def get_statuses(self) -> List[CommitFlag]:
         self.__update()
         return self.project.get_commit_statuses(self._raw_pr["commit_stop"])
+
+    def set_flag(
+        self,
+        username: str,
+        comment: str,
+        url: str,
+        status: Optional[CommitStatus] = None,
+        percent: Optional[int] = None,
+        uid: Optional[str] = None,
+    ) -> dict:
+        """
+        Set a flag on a pull-request.
+
+        See "Flag a pull-request" at
+
+            https://pagure.io/api/0/#pull_requests-tab
+
+        for a full description of the parameters.
+
+        :param username: The name of the application to be presented to users
+                         on the pull request page.
+        :param comment: A short message summarizing the presented results.
+        :param url: A URL to the result of this flag.
+        :param status: The status of the task.
+        :param percent: A percentage of completion compared to the goal.
+        :param uid: A unique identifier used to identify a flag on a pull-request.
+        :return: dict with the response received from Pagure.
```

Comment:
What about some class for that? (Or at least `CommitFlag` with some information not set?)

But we can create an issue and leave that for the future.

Reply:
Created #384.